src/Components/Test.jsx

```diff
@@ -0,0 +1,30 @@
+//This is a template Component for implementing dark mode using custom hooks 
```

Comment:
why two test utils? if the idea is to display how to use your hook try https://storybook.js.org/, for now, remove it from one of the places.

Reply:
sure